release-schema.json

```diff
@@ -98,6 +98,45 @@
             "null"
           ],
           "minLength": 1
+        },
+        "advancedElectronicSignatureRequired": {
+          "title": "Advanced electronic signature required",
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether an advanced or qualified electronic signature or seal (as defined in European Parliament and Council Regulation (EU) No 910/2014) is required."
```

Comment:
Put description after title.

Reply:
I think I ordered the keywords according to your suggestion in https://github.com/open-contracting/european-union-support/issues/173#issuecomment-1542949533. Is there a reason for different orders in different places?

Reply:
Oh, I meant the `type` field, not the `type` JSON Schema keyword.

Reply:
Ah, I see! Hopefully my misunderstanding didn't propagate too far!

Reply:
No worries! I'll correct it across extensions, looks like there's maybe just one new case in the statistics extension, and some pre-existing order differences.